atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SpecificTableSweeper.java

```diff
@@ -139,8 +139,8 @@ void runOnceForTable(TableToSweep tableToSweep,
             log.info("Swept successfully.",
                     LoggingArgs.tableRef("tableRef", tableRef),
                     UnsafeArg.of("startRow", startRowToHex(startRow)),
-                    SafeArg.of("unique cells swept", results.getCellTsPairsExamined()),
-                    SafeArg.of("deletion count", results.getStaleValuesDeleted()),
+                    SafeArg.of("cell+ts pairs examined", results.getCellTsPairsExamined()),
```

Comment:
`cell+ts` not sure how good it will be with regex searches :(. Can we make this cellTs, same everywhere?